### PR TITLE
Fixed screen sizing issue 20306

### PR DESCRIPTION
**Pull request by rarmitag**
*Issue #:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namex-dev--pr-1447-qexclmdh.web.app

File: pages/Examine.vue [hunk 1]
```diff
@@ -50,7 +50,7 @@
       v-if="
         !examine.isComplete && !reservedOrCondReserved && !examine.isEditing
       "
-      class="flex grow flex-col space-x-3 bg-gray-100 p-4 xl:flex-row"
+      class="flex grow flex-col space-x-3 bg-gray-100 p-4 2xl:flex-row"
     >
       <ExamineRecipe class="xl:w-1/2" />
       <ExamineDecision v-if="showDecisionPanel" class="xl:w-1/2" />
```
Comment (semmatti) [inline pages/Examine.vue:56]:
```suggestion
      <ExamineRecipe class="2xl:w-1/2" />
      <ExamineDecision v-if="showDecisionPanel" class="2xl:w-1/2" />
```

**Review by semmatti — APPROVED** [review]

**Pull request merged**